lib/app/index.js

```diff
@@ -66,11 +196,10 @@ exports.get = function (name) {
 /**
  * Returns all app instances in a list
```

Comment:
Indeed. I think that should be our default approach in all getter functions. You don't really wanna give something back that when manipulated without ill-intent would break the behavior. `slice` is also a very cheap operation in JS. I'm not sure why, but I suspect it's copy-on-write.